lib/action.js

```diff
@@ -79,7 +79,6 @@ function action(system) {
 	self.system.on('instance_save', function() {
 		setImmediate(function() {
 			self.io.emit('actions', self.actions);
-			self.io.emit('release_actions', self.release_actions);
```

Comment:
Is this not needed?

Reply:
My best guess is that originally there was a difference in what could be run between actions and release_actions.
All of the client or io emits removed here had no handlers I could find. I have added,removed and executed release_actions and havent found anything broken from this too

Reply:
I came across this when I was dealing with #1058, as I was noticing `self.release_actions` was never initialized like others in the file nor was it ever populated.  Makes sense because the actions and release actions use the same pool.  Mostly seems like the full boar copy/paste ended up with rouge code that was never cleaned up.